### ISSUE TITLE
PyTorch Training Stuck at Epoch 0

### ISSUE THREAD

**Issue by harshitsankhla** [xavysp/DexiNed#55]
After following the configuration mentioned in the README, my training loop is stuck at this point - 

_Mon Nov 16 01:53:58 2020 Epoch: 0 Sample 75/76 Loss: 30482.720703125_ 

And doesn't move any further, just stuck there.

**Comment by xavysp** [comment]
Hi, please wait for  one week more, we are working to solve that and other issues.

**Comment by harshitsankhla** [comment]
issue resolved, not a problem with the code. sorry for the false positive.
